src/Symfony/Component/Notifier/Bridge/Slack/SlackOptions.php

```diff
@@ -200,4 +200,19 @@ public function threadTs(string $threadTs): static
 
         return $this;
     }
+
+    public function getConversationsId(): ?string
+    {
+        return $this->options['conversations_id'] ?? null;
+    }
+
+    /**
+     * @return $this
+     */
+    public function conversations(string|array|callable $id): static
+    {
+        $this->options['conversations_id'] = $id;
```

Comment:
I would prefer to have this property an array instead of a string

Reply:
Well, If we make the type of this property an array, how do we store callable ?

Reply:
Not yet, but who knows? )
Ok, I'll remove the callable.